### ISSUE TITLE
Error reading dataset 2414

### ISSUE THREAD

**Issue by theScriptingEngineer** [ladisk/pyuff#64]
Hello,

when reading the sets in the example file '2411 and 2414.uff' an error occurs.

In order to reproduce:
```
import pyuff
uff_file = pyuff.UFF('../pyuff/data/2411 and 2414.uff')
#uff_file = pyuff.UFF('../pyuff/data/beam.uff')

typesInFile = uff_file.get_set_types()
print(typesInFile)

data: list[dict] = uff_file.read_sets()
```
Records 14 and 15 are repeated for each node, but I don't see this reflected in the code.

Thanks,
Frederik

**Comment by jankoslavic** [comment]
Please check:
https://github.com/ladisk/pyuff/tree/2411-tests
ttys

**Comment by theScriptingEngineer** [comment]
I traced the error to the _D_ notation of the exponent. The python function _float_ does not support this, only the letter e as an exponent.
I have fixed it in my fork of the project: FEASupport branch.
I believe there is an option to perform a pull request from my fork back to the original project.
[pyuff with added support for reading dataset 2414](https://github.com/theScriptingEngineer/pyuff)

Regards,
Frederik
